apps/dashboard/src/components/subscribers/subscriber-activity-drawer.tsx

```diff
@@ -0,0 +1,66 @@
+import { forwardRef } from 'react';
+import { VisuallyHidden } from '@radix-ui/react-visually-hidden';
+
+import { SheetContent, SheetDescription, SheetTitle } from '@/components/primitives/sheet';
+import { usePullActivity } from '@/hooks/use-pull-activity';
+import { Sheet } from '@/components/primitives/sheet';
+import { cn } from '@/utils/ui';
+import { ActivityPanel } from '@/components/activity/activity-panel';
+import { ActivitySkeleton } from '@/components/activity/activity-skeleton';
+import { ActivityLoadFailure } from '@/components/activity/activity-load-failure';
+import { ActivityHeader } from '@/components/activity/activity-header';
+import { ActivityOverview } from '@/components/activity/components/activity-overview';
+import { ActivityLogs } from '@/components/activity/activity-logs';
+
+type ActivityPanelDrawerProps = {
+  onActivitySelect: (activityId: string) => void;
+  activityId: string;
+};
+
+export const ActivityDetailsDrawer = forwardRef<HTMLDivElement, ActivityPanelDrawerProps>((props, ref) => {
```

Comment:
@LetItRock As far as I saw in the preview deployment it would be really cool to make this second drawer a bit shorter in width than the parent subscriber sheet. This will create the stack effect (similar to the new-relic UI).